dvc/remote/local.py

```diff
@@ -32,19 +32,11 @@ class RemoteLOCAL(RemoteBASE):
 
     DEFAULT_CACHE_TYPES = ["reflink", "copy"]
 
+    CACHE_MODE = 0o444
```

Comment:
Continuing using octal notation since it is much more readable than stat constants. Need to double check the windows though, since it uses some weird perms.

Reply:
Windows is fine.